plugin/__init__.py

```diff
@@ -918,6 +918,41 @@ def updateModelStyling(self, model):
         models.flush()
 
 
+    @util.api()
+    def findAndReplaceInModels(self, modelName, findText, replaceText, Front = True, Back = True, css = True):
+        if modelName == 'all_models':
```

Comment:
Just make `modelName` be optional; if not provided, it means all models.

Reply:
No I mean just make it `None`, this way if someone has a `modelName` of `all_models` it won't be an issue. There's no reason to make this be an actual special string. Also, I see that `findText` and `replaceText` default to empty string, why? I think if you are calling `findAndReplaceInModels` it's reasonable to assume that `findText` and `replaceText` are provided.

Reply:
Updated..!! Please check